homeassistant/components/intellifire/sensor.py

```diff
@@ -85,6 +86,42 @@ def _time_remaining_to_timestamp(data: IntellifirePollData) -> datetime | None:
         device_class=SensorDeviceClass.TIMESTAMP,
         value_fn=_time_remaining_to_timestamp,
     ),
+    IntellifireSensorEntityDescription(
+        key="downtime",
+        name="Downtime",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda data: data.downtime,
```

Comment:
Same.

Reply:
I attempted to put this into a timestamp - but it looks bad and I'll remove it unless ya'll have a fix.

Because the sensor value updates every 15 seconds or so the "downtime" seems to "tick" from

<img width="244" alt="image" src="https://user-images.githubusercontent.com/6491743/154678047-5014d2cf-86aa-44b2-a8c9-4acb7f08834c.png">

to 

<img width="262" alt="image" src="https://user-images.githubusercontent.com/6491743/154678088-42325d63-a2c2-4f10-8de5-7305f58c0de6.png">

and then resets back to "now".

I also tried dropping a `None` in when the value was `0` which didn't seem to fix stuff.



Switching it to a subtraction gives me:

<img width="268" alt="image" src="https://user-images.githubusercontent.com/6491743/154680228-fa793cfe-ed0e-4515-a780-7a17797219ed.png">

Is there any formatting options I can throw at this or is what I'm seeing just the way timestamps work?